code/API_definitions/simple-edge-discovery.yaml

```diff
@@ -212,26 +265,34 @@ paths:
         - name: IPv4-Address
           in: header
           required: false
-          description: The public IPv4 address allocated to the device.
+          description: The public IPv4 address allocated to the device by the network operator.
           example: "84.125.93.10"
           schema:
             $ref: "#/components/schemas/SingleIpv4Addr"
-
+            
+        - name: Public-port
+          in: header
+          required: false
+          description: The public TCP or UDP port allocated to the device by the network operator.
+          example: 123
+          schema:
+            $ref: "#/components/schemas/Port"
+            
         - name: IPv6-Address
           in: header
           required: false
-          description: The public IPv6 address allocated to the device.
+          description: The public IPv6 address allocated to the device by the network operator.
           example: "2001:db8:85a3:8d3:1319:8a2e:370:7348"
           schema:
             $ref: "#/components/schemas/DeviceIpv6Address"
 
         - name: Network-Access-Identifier
           in: header
+          required: false
           description: |
             3GPP network access identifier for the subscription
             being used by the device.
           example: "4d596ac1-7822-4927-a3c5-d72e1f922c94@domain.com"
```

Comment:
```suggestion
```

If I'm not wrong `example` parameter should be removed from NetworkAccessIdentifier object. Let me know if this is correct